From d6c8efdd0d0c9de9243f359f6ff32f2755ac320e Mon Sep 17 00:00:00 2001
From: Vagrant Cascadian <vagrant@debian.org>
Date: Wed, 25 Jul 2018 11:02:09 +0800
Subject: [PATCH 2/4] arm: odroid: Increase default env size in preparation for
 distro_bootcmd.

Adding distro_bootcmd support bumps the default environment size over
4500. Increase to SZ_16K to allow for room to grow in the future.

Acked-by: Lukasz Majewski <lukma@denx.de>

Series-changes: 2
- Change from 16384 to SZ_16K, as suggested by Minkyu Kang.
---
 include/configs/odroid.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/include/configs/odroid.h b/include/configs/odroid.h
index b7b75c0dc0..07882aca9a 100644
--- a/include/configs/odroid.h
+++ b/include/configs/odroid.h
@@ -50,7 +50,7 @@
 #define CONFIG_SYS_MONITOR_BASE	0x00000000
 
 #define CONFIG_SYS_MMC_ENV_DEV		CONFIG_MMC_DEFAULT_DEV
-#define CONFIG_ENV_SIZE			4096
+#define CONFIG_ENV_SIZE			SZ_16K
 #define CONFIG_ENV_OFFSET		(SZ_1K * 1280) /* 1.25 MiB offset */
 #define CONFIG_ENV_OVERWRITE
 
-- 
2.11.0

